salt/modules/tomcat.py

```diff
@@ -631,7 +631,7 @@ def deploy_war(war,
 
 def passwd(passwd,
            user='',
-           alg='md5',
+           alg='sha1',
```

Comment:
I was not sure if this is OK for Tomcat, since this module replaces that there. SHA1 OK but others I didn't looked up if this is compatible.